app/retail/templates/emails/grants/transaction_summary.html

```diff
@@ -0,0 +1,224 @@
+    float: left;
+  }
+
+  .funding {
+    width: 100%;
+  }
+
+  .leftFunding {
+    width: 50%;
+    float: left;
+    text-align: left;
+  }
+  
+  .rightFunding {
+    width: 50%;
+    float: left;
+    text-align: right;
+  }
+
+  .funding>div>h4 {
+    line-height: 0px;
+    color: #0D0764;
+    padding-top: -50px;
+  }
+
+  .leftFunding>h4 {
+    text-align: left;
+  }
+
+  .rightFunding>h4 {
+    text-align: right;
+  }
+
+  .progressBarLine {
+    width: 1px;
+    height: 45px;
+    background-color: #979797;
+    float: left;
+    transform: translateY(-20%);
+    margin-right: 20px;
+  }
+
+  .funding>div>p {
+    line-height: 0px;
+    color: #BBBDBF;
+  }
+
+  table {
+    width: 100%;
+  }
+
+  td:nth-child(1)>img {
+    border-radius: 25px;
+  }
+
+</style>
+<img id="newSupporter" src="{% static "v2/images/transaction-summary.png" %}" alt="Transaction Summary" title="Transaction Summary">
+<h1 style="text-transform: none;">{% trans "Grant" %} '{{ grant_name }}' {% trans "has received more funding" %}!</h1>
+<br>
+
+<hr>
+<div class="grantInfo">
+  <a href="https://github.com/{{ github_username }}">
+    <img src="{{ github_username }}" />
+  </a>
+  <!-- width="50" height="50"Input project image here -->
+  <br>
+  <a class="grantInfoName" href="https://github.com/{{ github_username }}">{{ github_username }}</a>
+  <!-- Input project name here -->
+  <p>{{ grant_description }}</p>
+  <!-- Input project description here -->
+</div>
+<hr>
+<div class="progress" id="tablePadding">
+  <h3 style="text-transform: none;"><strong>{% trans "Funding Recieved in the Last 24 Hours" %}</strong></h3>
+  <!-- Content in table should be pre-generated with JS. -->
+  <table>
+    <tr>
+      <td><img src="{{ funder_i_username }}" /></td>
+      <td><p>{{ funder_i_username }}</p></td>
+      <td><p><strong>{{ funder_i_amount }}</strong></p></td>
+      <td><p>{{ funder_i_timestamp }}</p></td>
+      <td><a href="{{ funder_i_link }}" style="color: #4C33AB;"><strong>View Transaction</strong></a></td>
+    </tr>
+  </table>
+</div>
+<hr>
+<div class="progress">
+  <h3><strong>{% trans "Total Funds" %}</strong></h3>
+  <h3 style="text-transform: none;">{{ funding_status }}</h3>
+  <div class="progressBar">
+    <div class="progressCompleted" style="width: {{ progress_completed }}%;">
+      <!-- Dynamically change the width of your progressCompletion, ie 50% -->
+
+    </div>
+  </div>
+  <div class="funding">
+    <div class="leftFunding">
+      <h4>
+        <strong>{{ current_value }}</strong>
+      </h4>
+      <!-- Input current funding value -->
+      <p>{%trans "Current" %}</p>
+    </div>
+    <div class="rightFunding">
+      <h4>
+        <strong>{{ goal_value }}</strong>
+      </h4>
+      <!-- Input goal funding value -->
+      <p>{%trans "Goal" %}</p>
+    </div>
+
+  </div>
+</div>
+<hr>
+
+<br>
+<br>
+<a class="updatesButton" href="https://gitcoin.co/grants/{{ grant_link }}">{%trans "View Grant" %}</a>
```

Comment:
This should use `url` vs a hardcoded path.

Reply:
In the latest commit, I've pushed it out as a relative var, but I assume you want me to use a `{% url %}` as found in the `urls.py` file correct?

So it would be something like `href="{% url grants.views.grant_details %}"`?